pkg/sql/catalog/systemschema/system.go

```diff
@@ -1056,13 +1057,15 @@ const SystemDatabaseName = catconstants.SystemDatabaseName
 // descriptor.
 func MakeSystemDatabaseDesc() catalog.DatabaseDescriptor {
 	priv := privilege.List{privilege.CONNECT}
+	binaryVersion := clusterversion.ByKey(clusterversion.BinaryVersionKey)
```

Comment:
I think init'ing this to binary version will mean we can't bump it in any permanent migrations, and is maybe a problem for when we bootstrap to the old schema that was as of minBinary, e.g. during tenant creation. I think we want to plumb in the system schema version that we're bootstrapping right?

Reply:
I think another thing we might want to do is make a var somewhere -- maybe in systemschema -- that anyone who adds a bumping migration then bumps, and use that, rather than clusterversion.BinaryVersionKey, for the default value during bootstrap. This will mean one more step for migration-that-bumps-it authors, but importantly means folks who add a CV _that doesn't bump the system schema version_ don't need to regenerate all the testdata / catalog dumps / etc?